core/lib/spree/money.rb

```diff
@@ -4,6 +4,8 @@
 require 'monetize'
 require 'active_support/core_ext/string/output_safety'
 
+Money.locale_backend = :i18n
```

Comment:
I think we should not add global configuration in a class like this.

Could we move this into it's own `config/initializers/money.rb` of the host app instead? And tell users to do so in their current apps and add it into the install generator? I like to be explicit with these things, especially if it's not our gem we are configuring here.

Reply:
I agree this should be done in a more configurable manner, but I don’t think we should require everyone to update their apps with an initializer by default. I’m thinking just allow apps to override the setting if they really want to. WDYT?

Reply:
I’m not a fan of adding a setting in our app that’s a proxy to a foreign gem. 

What about putting it inside a `before_initialize` block? Are engine `config/initialize` files read before host apps files? That we be preferable. Someway where the host app is able to set it in a default (read: mentioned in money gems readme) kind of way and we still act as fallback/default?